kitsune/wiki/jinja2/wiki/revision.html

```diff
@@ -1,11 +1,11 @@
 {% extends "wiki/base.html" %}
 {% from "wiki/includes/sidebar_modules.html" import document_tools with context %}
-{% from "wiki/includes/document_macros.html" import show_for with context %}
 {# L10n: {n} is the revision number, {t} is the title of the document. {c} is the category. #}
 {% set title = _('Revision {n} | {t} | {c}')|f(n=revision.id, t=document.title, c=document.get_category_display()) %}
 {% set crumbs = [(document.get_absolute_url(), document.title),
                  (None, _('Revision {num}')|f(num=revision.id))] %}
 {% set classes = 'document' %}
+{% set scripts = ('document',) %}
```

Comment:
Does it makes sense to add a revision.js file which will import showFor and initialize it? Then we can use just that file

Reply:
@akatsoulas Yes, I think that's better. I'll do that.